sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/SamplingStrategyResponse.java

```diff
@@ -164,13 +164,13 @@ Builder setSamplingStrategyType(SamplingStrategyType samplingStrategyType) {
     Builder setRateLimitingSamplingStrategy(
         RateLimitingSamplingStrategy rateLimitingSamplingStrategy) {
       this.rateLimitingSamplingStrategy = rateLimitingSamplingStrategy;
-      return this;
+      return setSamplingStrategyType(SamplingStrategyType.RATE_LIMITING);
```

Comment:
Can you add a comment on why we need to do this?

Reply:
Done